core/src/main/java/com/amplifyframework/core/Amplify.java

```diff
@@ -127,6 +128,11 @@ public static void configure(@NonNull final AmplifyConfiguration configuration,
                 );
             }
 
+            boolean debuggableApp = isDebuggableApplication(context);
+            if (debuggableApp && Logging.getPlugins().isEmpty()) {
+                addPlugin(new AndroidLoggingPlugin());
+            }
+
```

Comment:
`Amplify` is a "façade." A downside to the façade as a pattern is that it can become a god object coupled to all classes of an app. [ref](https://refactoring.guru/design-patterns/facade).

We want to "fan out" quickly. Strive for a ~one-line integration of your Dev Menu code.

Would it be possible to have your entire integration be a single line?
```java
DevMenu.attach(this);
```

All of your logic, like whether or not its a debug build, whether or not you need a logging plugin, etc. etc. would then be in your `DevMenu`, not in `Amplify`.